backend/app/rest/api/rest.go

```diff
@@ -31,6 +31,28 @@ import (
 	"github.com/umputun/remark/backend/app/store/service"
 )
 
+// sslMode defines rest server mode (http or https)
```

Comment:
probably moving all this chunk to a separate file (ssl.go, for example) will make some sense. At the same time functions related to ssl, like make HttpsServer amd httpToHTTPSRouter can go to the same file.

Reply:
Ok, done )